src/Sentry.Unity/SentryUnitySdk.cs

```diff
@@ -137,4 +125,77 @@ public void CaptureFeedback(string message, string? email, string? name, bool ad
 
         Sentry.SentrySdk.CurrentHub.CaptureFeedback(message, email, name, hint: hint);
     }
+
+    internal static void HandleWindowsPlayer(SentryUnitySdk unitySdk, SentryUnityOptions options)
```

Comment:
Not sure this method is called. "Handle"? It seems it's "Setting Up"? or Initializing?

It's being called by `Init` so perhaps `InitWindowsPlayer`?